[2025-03-26T03:34:07.827+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-26T03:34:07.901+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-26T01:30:00+00:00 [queued]>
[2025-03-26T03:34:07.938+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-26T01:30:00+00:00 [queued]>
[2025-03-26T03:34:07.941+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2025-03-26T03:34:08.022+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_all_data> on 2025-03-26 01:30:00+00:00
[2025-03-26T03:34:08.053+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-26T03:34:08.064+0000] {standard_task_runner.py:72} INFO - Started process 91 to run task
[2025-03-26T03:34:08.083+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'get_all_data', 'scheduled__2025-03-26T01:30:00+00:00', '--job-id', '207', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmpbl_d3kx0']
[2025-03-26T03:34:08.099+0000] {standard_task_runner.py:105} INFO - Job 207: Subtask get_all_data
[2025-03-26T03:34:08.321+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-26T01:30:00+00:00 [running]> on host b3fc939aa2c6
[2025-03-26T03:34:08.561+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='get_all_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-26T01:30:00+00:00'
[2025-03-26T03:34:08.568+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-26T03:34:08.604+0000] {inaturalist_dag.py:150} INFO - Getting data from keyword: atropelamento
[2025-03-26T03:34:08.627+0000] {inaturalist_dag.py:41} INFO - Maximum id 264497584 for the request
[2025-03-26T03:34:09.491+0000] {inaturalist_dag.py:58} INFO - Got 0 results
[2025-03-26T03:34:09.495+0000] {inaturalist_dag.py:158} INFO - No more observations found, stopping.
[2025-03-26T03:34:09.527+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-26T03:34:09.537+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-03-26T03:34:09.586+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-26T03:34:09.586+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=get_all_data, run_id=scheduled__2025-03-26T01:30:00+00:00, execution_date=20250326T013000, start_date=20250326T033407, end_date=20250326T033409
[2025-03-26T03:34:09.648+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-26T03:34:09.694+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-26T03:34:09.697+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
